test/Integration/SqlOutputBindingIntegrationTests.cs

```diff
@@ -162,7 +161,6 @@ public void QueueTriggerProductsTest(SupportedLanguages lang)
 
         [Theory]
         [SqlInlineData()]
-        [UnsupportedLanguages(SupportedLanguages.OutOfProc)]
         public void TimerTriggerProductsTest(SupportedLanguages lang)
```

Comment:
Did we already have TimerTriggerProducts for OOP?

Reply:
yes we did, the sample and test were working on my local but were failing on the pipeline. I realized that's probably because of the negative scenarios overwriting the samples.